R/db-hive.R

```diff
@@ -0,0 +1,43 @@
+#' @export
+db_desc.Hive <- function(x) {
```

Comment:
Can you just make this a generic for the ODBC connection class? There's no need copy and paste the same code for each driver.